src/beam.ts

```diff
@@ -555,6 +556,8 @@ export class Beam extends Element {
       // iterate through notes, calculating y shift and stem extension
       for (let i = 1; i < notes.length; ++i) {
         const note = notes[i];
+        // Skip Ghostnotes
+        if (note instanceof GhostNote) continue;
```

Comment:
Can we remove these checks? There must be a better abstraction here, no?

Reply:
Beams accept StemmableNotes and GhostNote extends StemmableNote. The abstraction to make them Beam ignore GhostNotes based on the calculations within the for loop would be much less obvious. Would you have a proposal? Today if we leave a GhostNote in the middle of StaveNotes belonging to a Beam, vexflow will not handle it correctly. I believe that this a case to handle even if GhostNotes are not supported by Easyscore.

Reply:
I think the right fix here is for GhostNote not to be stemmable. Or alternatively, `hasStem()` can always return false for GhostNotes -- so in here, you can ignore any notes that don't have stems. Maybe that will work.

Reply:
Now with `hasStem()`